content/10-minute-tutorial.md

```diff
@@ -1,3 +1,328 @@
+
+2. Select **New > Java Class** and give the class a name (for instance, `StepDefinitions.java`)
+
+3. Paste in the snippets
+
+IntelliJ will not automatically recognise those symbols (like `@Given`, `@When`, `@Then`), so we'll need to add `import` statements. In IntelliJ:
+
+4. Put your cursor on the `@Given` symbol and press **ALT** + **ENTER**, then select **Import class**.
+
+5. Do the same for the other symbols (underlined in red).
+
+Now, when you run the test, your step definitions will be found and run.
+
+{{% note "Run configurations"%}}
+If this does not work, select **Run > Edit Configurations**, select **Cucumber java** from the **Defaults** drop-down, and add the project name to the **Glue** field on the **Configuration** tab.
+{{% /note %}}
+
+Your result will include something like the following:
+```
+cucumber.api.PendingException: TODO: implement me
+	at skeleton.Stepdefs.i_have_cukes_in_my_belly(Stepdefs.java:10)
+	at ✽.I have 42 cukes in my belly(/Users/maritvandijk/IdeaProjects/cucumber-java-skeleton/src/test/resources/skeleton/belly.feature:4)
+
+```
+
+The reason for this is that we haven't actually implemented this step; it throws a `PendingException` telling you to implement the step.
+We will need to implement all steps to actually do something.
+
+## Implement the steps
+
+We will need to implement all steps to actually do something.
+
+1. Update your `StepDefinitions.java` class to implement the step definition.
+
+The step can be implemented like this:
+```java
+    @Given("^I have (\\d+) cukes in my belly$")
+    public void I_have_cukes_in_my_belly(int cukes) throws Throwable {
+        Belly belly = new Belly();
+        belly.eat(cukes);
+    }
+```
+
+To make this step compile we also need to implement a class Belly with a method eat().
+
+2. Implement the class `Belly.java` inside your `src/main/java/<project>` folder; create your `<project>` directory here (if needed)
+
+Now you run the test and implement the code to make the step pass. Once it does, move on to the next step and repeat!
+
+## Result
+
+Once you have implemented all your step definitions (and the expected behaviour in your application!) and the test passes, the summary of your results should look something like this:
+
+```
+Tests run: 5, Failures: 0, Errors: 0, Skipped: 4, Time elapsed: 0.656 sec
+
+Results :
+
+Tests run: 5, Failures: 0, Errors: 0, Skipped: 4
+
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time: 5.688 s
+[INFO] Finished at: 2017-05-22T15:43:29+01:00
+[INFO] Final Memory: 15M/142M
+[INFO] ------------------------------------------------------------------------
+```
+
+# Examples
+If you prefer to get started with a working project, try the skeleton project [which is available from GitHub](https://github.com/cucumber/cucumber-java-skeleton).
```

Comment:
I'd link "skeleton project" not "available on GitHub"

Reply:
And by this stage, asking them if they prefer something else feels odd. Maybe "If you'd like to test out a working example, try...